pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -64,9 +64,7 @@ export const METRIC_CATALOGUE = {
   },
   blk_read_time: {
     humanizeName: 'Reading Blocks Time',
-    tooltipText:
-      'Total time the statement spent reading blocks, in'
-      + ' milliseconds (if track_io_timing is enabled, otherwise zero)',
+    tooltipText: 'Total time (in milliseconds) the statement spent reading blocks (if track_io_timing is enabled, otherwise zero)',
```

Comment:
@lunaticusgreen , @BupycHuk  do we really use milliseconds here or we converted them to some other units?
+ is this needed to have units in the description, if we convert units on the UI anyway?

Reply:
We convert them to seconds.